x-pack/plugins/snapshot_restore/public/application/constants/index.ts

```diff
@@ -112,3 +114,18 @@ export const UIM_POLICY_CREATE = 'policy_create';
 export const UIM_POLICY_UPDATE = 'policy_update';
 export const UIM_RETENTION_SETTINGS_UPDATE = 'retention_settings_update';
 export const UIM_RETENTION_EXECUTE = 'retention_execute';
+
+// DisableToolTip messages
+export const MANAGED_REPOSITORY_TOOLTIP_MESSAGE = i18n.translate(
```

Comment:
I gave it a second thought and maybe these messages would be a better fit in `x-pack/plugins/snapshot_restore/public/application/components/disable_tooltip.tsx` since they would most likely be used only for the `DisableToolTip` component so it would make more sense to keep them together with the component. Apologies for the back-and-forth on this!